File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-06/plagiarized/L6/04/Kasus6.kt
package IR

/**
 * @author C166AB8FBBBE55174E16A67530426A8A 19011F43894F06E2BA0CA74061427469
 */
object Kasus6 {
    
    fun simpanArray(arr: IntArray, i: Int) {
        val sc = java.util.Scanner(System.`in`)
        
        if (i > 0) {
            simpanArray(arr, i - 1)
            println("Read a number: ")
            arr[i - 1] = sc.nextInt()
        }
        
    }
    
    fun tampilArray(arr: IntArray, i: Int) {
        if (i >= 0) {
            println(arr[i])
            tampilArray(arr, i - 1)
        }
    }
    
    @JvmStatic
    fun main(args: Array<String>) {
        val n = 10
        val arr = IntArray(n)
        simpanArray(arr, n)
        tampilArray(arr, n - 1)
    }
}

File: datasets/IR-Plag/convertai-original-java-rest-python/case-06/plagiarized/L5/04/Kasus6.py
import sys

def simpanArray(arr, n):
    sc = sys.stdin
    for i in range(n):
        print("Read a number: ", end="")
        arr[i] = int(sc.readline().strip())

def tampilArray(arr, i):
    print(arr[i])

if __name__ == "__main__":
    n = 10
    arr = [0] * n
    simpanArray(arr, n)
    for i in range(n-1, -1, -1):
        tampilArray(arr, i)

